node/network/approval-distribution/src/lib.rs

```diff
@@ -41,19 +41,19 @@ use polkadot_node_subsystem::{
 };
 use polkadot_node_subsystem_util::metrics::{self, prometheus};
 use polkadot_node_network_protocol::{
-	PeerId, View, v1 as protocol_v1, ReputationChange as Rep,
+	PeerId, View, v1 as protocol_v1, ReputationChange, UnifiedReputationChange as Rep,
+
 };
 
 const LOG_TARGET: &str = "approval_distribution";
 
-const COST_UNEXPECTED_MESSAGE: Rep = Rep::new(-100, "Peer sent an out-of-view assignment or approval");
-const COST_DUPLICATE_MESSAGE: Rep = Rep::new(-100, "Peer sent identical messages");
-const COST_ASSIGNMENT_TOO_FAR_IN_THE_FUTURE: Rep = Rep::new(-30, "The vote was valid but too far in the future");
-const COST_INVALID_MESSAGE: Rep = Rep::new(-1000, "The vote was bad");
-
-const BENEFIT_VALID_MESSAGE: Rep = Rep::new(10, "Peer sent a valid message");
-const BENEFIT_VALID_MESSAGE_FIRST: Rep = Rep::new(15, "Valid message with new information");
+const COST_UNEXPECTED_MESSAGE: Rep = Rep::MinorAnnoy("Peer sent an out-of-view assignment or approval");
+const COST_DUPLICATE_MESSAGE: Rep = Rep::MinorAnnoy("Peer sent identical messages");
+const COST_ASSIGNMENT_TOO_FAR_IN_THE_FUTURE: Rep = Rep::MinorAnnoy("The vote was valid but too far in the future");
```

Comment:
this should arguably be lower then `COST_UNEXPECTED_MESSAGE`

Reply:
The general Q here is more on the lines of how many quantizations we need. I am fine with adding another level `Miniscule*` to the possible options.